### DISCUSSION TITLE
AIR - Filters Initialization

### DISCUSSION THREAD

**Discussion by AhmedThahir** [AhmedThahir/AhmedThahir.github.io#113] [Q&A]
I'm self-studying AI in Robotics by Dr. Ram Karthikeyan. I'm currently in the Filters module.

Wouldn't it be best to always initialize the system as 0? That would be most unbiased estimate, right? This way when we do the first iteration of the update step, the first measurement would be given full weightage.

**Comment by AhmedThahir** [comment]
@Fir121 How can I edit this to make it appear for AIR?

This is the exact path: https://uni-notes.netlify.app/cs_electives/ai_in_robotics/02_filters/

**Comment by Fir121** [comment]
Can't do that, your comment needs to be in the thread not in the discussion heading. Just copy this into a new comment on the page instead, it will auto create the discussion. You can delete this after if you want.

**Reply by AhmedThahir** [reply]
Why can't we do it in reverse?

I don't get it

**Reply by AhmedThahir** [reply]
We can't create a discussion from
GitHub and get it picked by giscus?

**Reply by Fir121** [reply]
You can but you comment is on the top (red arrow) not in the comments area (yellow arrow) so it wont be displayed.

![image](https://github.com/uni-notes/uni-notes/assets/62911996/6e1445ba-aab6-42bd-8b9c-6e3c2d8608de)

**Reply by AhmedThahir** [reply]
Will the latest one appear?

**Reply by Fir121** [reply]
Yeah it will, you will need to move this discussion to the comments category and change the title to cs_electives/ai_in_robotics/02_filters

OR the more direct way is just use the comment box but yeah.

**Reply by AhmedThahir** [reply]
Alright!
Just wanted to know how to do it. Thanks :)

**Comment by AhmedThahir** [comment]
I'm currently on the Filters module. Wouldn't it be best to always initialize the system as 0? That would be most unbiased estimate, right? This way when we do the first iteration of the update step, the first measurement would be given full weightage.
